### ISSUE TITLE
Vnc

### ISSUE THREAD

**Issue by sam-pandey** [megatron0000/heroku-vnc#4]
Can i access this by vnc viewer cause keyboard doesn't work on phone

**Comment by agamya-samuel** [comment]
yes, @sam-pandey you can access it thru VNC viewer, if you deployed on heroku, just enter your app URL in the place of IP address of your NoVNC app, (you might also need to enter port number also) this should work,

**Comment by sam-pandey** [comment]
> yes, @sam-pandey you can access it thru VNC viewer, if you deployed on heroku, just enter your app URL in the place of IP address of your NoVNC app, (you might also need to enter port number also) this should work,

I tried but didn't get working port.
what can that be? I had tried 5901,5900,6000 and web port 443 too.

**Comment by agamya-samuel** [comment]
> > yes, @sam-pandey you can access it thru VNC viewer, if you deployed on heroku, just enter your app URL in the place of IP address of your NoVNC app, (you might also need to enter port number also) this should work,
> 
> I tried but didn't get working port.
> what can that be? I had tried 5901,5900,6000 and web port 443 too.

maybe it doesn't support it then @sam-pandey

**Comment by sam-pandey** [comment]
> > > yes, @sam-pandey you can access it thru VNC viewer, if you deployed on heroku, just enter your app URL in the place of IP address of your NoVNC app, (you might also need to enter port number also) this should work,
> > 
> > 
> > I tried but didn't get working port.
> > what can that be? I had tried 5901,5900,6000 and web port 443 too.
> 
> maybe it doesn't support it then @sam-pandey

I used ngrok for port forwarding 
It's working now on vnc viewer
